client/src/site-search/search-results.tsx

```diff
@@ -282,98 +308,138 @@ function Results({
   metadata: Metadata;
   suggestions: Suggestion[];
 }) {
+  const hitCount = metadata.total.value;
   const locale = useLocale();
+  const page = metadata["page"];
   const [searchParams] = useSearchParams();
+  const query = searchParams.get("q");
 
   return (
-    <div>
-      <div className="search-results">
-        <p>
-          Found <ShowTotal total={metadata.total} /> in {metadata.took_ms}{" "}
-          milliseconds.
-        </p>
-
-        {!!suggestions.length && (
-          <div className="suggestions">
-            <p>Did you mean...</p>
-            <ul>
-              {suggestions.map((suggestion) => {
+    <>
+      {hitCount > 0 ? (
+        <>
+          <h1>
+            Search results for, <span className="query-string">{query}</span>
+            {page && page !== 1 && `. Page ${page}`}
+          </h1>
```

Comment:
Thinking around this again, do we know how many pages of results there are?
I was thinking we could then have something like: "Page 2 of 10" and even on the first page of results we can then have, "Page 1 of 10" for example.

Reply:
So then the entire phrase would be something like:

"Search results for, box-shadow css. Page 1 of 10.", "Search results for, box-shadow css. Page 2 of 10." etc.